drivers/ipcc/ipcc_read.c

```diff
@@ -234,6 +234,7 @@ ssize_t ipcc_read(FAR struct file *filep, FAR char *buffer,
        * we don't have to do it.
        */
 
```

Comment:
If `leave_critical_section()` is added here, then comment above is stale.

Reply:
Thank you, I will remove the commit